tasks/build-css.js

```diff
@@ -333,8 +332,6 @@ gulp.task('build-css:concat-standalone-lg', function() {
       'dist/spectrum-' + colorStop + '.css'
     ])
       .pipe(concat('spectrum-' + colorStop + '-lg.css'))
-      // Replace instances of & that refer to the colorstop selector with .secptrum
-      .pipe(replace(/^&/gm, '.spectrum'))
```

Comment:
This is no longer needed as the generated file already has `&` replaced by `.spectrum`.